src/Symfony/Component/Routing/Loader/schema/routing/routing-1.0.xsd

```diff
@@ -93,6 +94,11 @@
     <xsd:attribute name="stateless" type="xsd:boolean" />
   </xsd:complexType>
 
+  <xsd:complexType name="resource">
+    <xsd:attribute name="path" />
+    <xsd:attribute name="namespace" />
```

Comment:
shouldn't we allow any attribute instead ? Other kind of loaders may want to use other attributes

Reply:
I thought listing the attributes here would be nicer for autocompletion. But you're right, I should probably allow arbitrary attributes here.